<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OffsetMenuForm</class>
 <widget class="QWidget" name="OffsetMenuForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>115</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройка сдвига</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="offsetLabel">
     <property name="text">
      <string>Сдвиг, м:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="zLayout">
     <item>
      <widget class="QLabel" name="zlabel">
       <property name="text">
        <string>Z:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="zDBox">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>100000000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="zSlider">
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="xLayout">
     <item>
      <widget class="QLabel" name="xLabel">
       <property name="text">
        <string>X:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="xDBox">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>50000000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="xSlider">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
